src/pages/posters.astro

```diff
@@ -0,0 +1,117 @@
+
+const allTracks = Array.from(
+  new Set(sessions.map((session) => session.data.track)),
+).sort();
+
+const allTypes = Array.from(
+  new Set(sessions.map((session) => session.data.submission_type)),
+).sort();
+
+const title = "Sessions";
+
+const description = "A list of all the sessions at the conference";
+---
+
+<Layout title={title} description={description}>
+  <div class="px-6">
+    <Prose>
+      <h1>Posters</h1>
+
+      <p>
+      During the lunch breaks, on Wednesday, Thursday and Friday we will host {posters.length} poster sessions. You can see them all below.
+      </p>
+
+      <p>
+      The Posters will be available throughout the entire event. If you want to talk to the author of the poster, please visit the expo area during the scheduled poster session.
+      </p>
+
+
+      <h2>List of posters</h2>
+
+    </Prose>
+
+    <ol class="sessions">
+      {
+        posters.map((session) => (
+          <li
+            class="mb-12"
+            data-track={session.data.track}
+            data-type={session.data.submission_type}
+          >
+            <Prose>
+              <h4>
+                <a href={`/session/${session.slug}`} class="underline">
+                  {session.data.title}
+                </a>
+              </h4>
+
+              <p>
+                <span class="sr-only">Speakers:</span>
+
+                <SessionSpeakers speakers={session.data.speakers} />
+              </p>
+            </Prose>
+
+            <ul class="space-x-2 capitalize">
+              <li class="inline-block">
+                <Tag>
+                  <span class="sr-only">Track:</span>
+                  {session.data.track}
+                </Tag>
+              </li>
+
+              <li class="inline-block">
+                <Tag>
+                  <span class="sr-only">Level:</span>
+                  {session.data.level}
+                </Tag>
+              </li>
+            </ul>
+          </li>
+        ))
+      }
+    </ol>
+  </div>
+</Layout>
+
+<script>
+  const forms = document.querySelectorAll("form.filter-sessions");
+
+  forms.forEach((form) => {
+    form.querySelectorAll("select").forEach((select) => {
+      select.addEventListener("change", () => {
+        // @ts-ignore
+        const track = form.querySelector("#track").value;
+        // @ts-ignore
+        const type = form.querySelector("#type").value;
+
+        document.querySelectorAll("ol.sessions > li").forEach((session) => {
+          const sessionTrack = session.getAttribute("data-track");
+          const sessionType = session.getAttribute("data-type");
+
+          if (
+            (track === "" || sessionTrack === track) &&
+            (type === "" || sessionType === type)
+          ) {
+            // @ts-ignore
+            session.style.display = "block";
+          } else {
+            // @ts-ignore
+            session.style.display = "none";
+          }
+        });
+      });
+    });
+  });
+</script>
```

Comment:
this can be removed :)

maybe we can turn this whole page in a component, to reuse the rest of the code